src/System.Device.Gpio/System/Device/Gpio/Drivers/RaspberryPiComputeModule3Driver.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace System.Device.Gpio.Drivers
+{
+	public class RaspberryPiComputeModule3Driver : RaspberryPi3Driver
+	{
+		protected internal override int PinCount => 46;

```

Comment:
Is the only difference between the ComputeModule3 driver and the raspberryPi driver the number of pins it exposes? If so, I'm not sure if this would grant the requirement of adding a whole new driver for it.